lsst-epo/investigations-client
# [C] allow number, text, and textarea in table cell

## What this change does ##

Allows standalone text, textarea, and number type questions to be added to a display table

## Notes for reviewers ##

Includes client and API branch.

Include an indication of how detailed a review you want on a 1-10 scale.
- 4

## Testing ##

- Create a mix of text, textarea, number, and table type questions
- Create a display table (within the investigation pages, not as a question)
- Create a row of headers to label the questions
- Create a row and add your previously created questions under "previous question"
    - the order added is the order the questions will be displayed

See that:
- The results of the questions are rendered in a read-only format with previously entered answers retrieved
- A mix of questions including mixing standalone questions and table type questions co-exist on single row

## Checklist ##

If any of the following are true, please check them off

## Screenshots (optional) ##
### Before:
### After:
